aiven/client/cli.py

```diff
@@ -44,7 +45,7 @@
 
 def convert_str_to_value(schema: Mapping[str, Any], str_value: Optional[str]) -> Any:
     if str_value is not None:
-        if "string" in schema["type"]:
+        if "string" in schema["type"] or "object" in schema["type"]:
```

Comment:
Though this seems a bit weird, is it now so that `str_value` can also be of object type? If so, I'd like to remove the `str_` prefix from name and have an accurate type hint for the arg.